Scripts/Hooks/scratchpad-hook.lua

```diff
@@ -796,8 +802,15 @@ local function loadScratchpad()
                 addCoordinateListener = function(listener)
                     table.insert(coordListeners, listener)
                 end,
+                addFrameListener = function(name, listener)
+                    frameListeners[name] = listener
```

Comment:
Any particular reason to use named events here compared to `missionLoadEndListeners` which is just a list?